### PR TITLE
Set initial block range for a newly created account (again)

### PR DESCRIPTION
**Pull request by rasom**
status: ready

### CODE REVIEW

**Review by flexsurfer — APPROVED** [review]

File: src/status_im2/contexts/profile/login/events.cljs [hunk 1]
```diff
@@ -132,19 +132,16 @@
...
   (if error
     {:db (update db :profile/login #(-> % (dissoc :processing) (assoc :error error)))}
-    (let [{:keys [creating?]} (:profile/login db)]
-      (rf/merge cofx
-                {:db         (dissoc db :profile/login)
-                 :dispatch-n [[:logging/initialize-web3-client-version]
-                              (when (and creating? (not recovered-account?))
-                                [:wallet/set-initial-blocks-range])]}
-                (if (or creating? recovered-account?)
-                  (profile.create/login-new-profile recovered-account?)
```
Comment (rasom) [inline src/status_im2/contexts/profile/login/events.cljs:146]: It looks like `(profile.create/login-new-profile recovered-account?)` was dead as both `creating?` and `recovered-account?` would be always `nil` nowadays, so I just removed it.

**Review by ibrkhalil — APPROVED** [review]

**Comment by pavloburykh** [comment]
@rasom thanx for the PR! 

Please take a look at the following issue

### ISSUE 1 App crashes on recovery via seed phrase (Android)

**Steps:** 

1. Proceed with recovering user via seed phrase
2. Observe crash on Generating keys step

**Actual result:** app crashes. 

In some cases different errors appearing: 

```
[5513,"RCTView",251,{"flexDirection":"row","marginHorizontal":"<<NaN>>","justifyContent":"space-between","marginBottom":16}] is not usable as a native method argument
```
[Status-debug-logs (44).zip](https://github.com/status-im/status-mobile/files/12250697/Status-debug-logs.44.zip)

```
Exception in HostFunction: Malformed calls from JS: field sizes are different.

[[2,2,2,2,2,2,2,2,2,2,2,2,2,26],[21,21,21,21,21,21,21,21,21,21,21,21,21,12],[[27,"RCTView",{"width":["window-width"]}],[39,"RCTView",{"width":["window-width"]}],[53,"RCTView",{"width":["window-width"]}],[69,"RCTView",{"width":["window-width"]}],[83,"RCTView",{"width":["window-width"]}],[95,"RCTView",{"width":["window-width"]}],[107,"RCTView",{"width":["window-width"]}],[1]],2571]
```
[Status-debug-logs (45).zip](https://github.com/status-im/status-mobile/files/12250700/Status-debug-logs.45.zip)

```
com.facebook.react.bridge.JSApplicationillegalArgumentException: Error while updating property 'width' in shadow node of type: RCTView
```

photo_2023-08-03 15 25 17

https://github.com/status-im/status-mobile/assets/97245802/7b7d4ff9-1d74-451f-a687-4cf0a2096411

Sometimes app crashes without any errors.

**Comment by rasom** [comment]
@pavloburykh and none of this happened in develop before right? And iOS is fine?

**Comment by pavloburykh** [comment]
> @pavloburykh and none of this happened in develop before right? And iOS is fine?

1. Have not faced those issues in develop
2. IOS is not crashing but still it is not fine. 

As you can see on video, in case of IOS after keys generation step the rest 3 steps (identifiers, notifications, welcome screens) are skipped and user is being redirected right into account


https://github.com/status-im/status-mobile/assets/97245802/9e2b6a9d-121a-4f75-a1ec-c1d498ed7a2a

**Comment by rasom** [comment]
@pavloburykh should be fixed now

**Comment by pavloburykh** [comment]
@rasom ready for merge.

**Pull request merged**